src/media/js/elements/nav.js

```diff
@@ -0,0 +1,182 @@
+define('elements/nav',
+    ['core/log', 'core/z'],
+    function(log, z) {
+
+    var logger = log('elements/nav');
+    console.log('initializing');
+
+    var MktNavElement = document.registerElement('mkt-nav', {
+        prototype: Object.create(HTMLElement.prototype, {
+            // Toggle the class specified by this.statusVisibleClass on
+            // this.statusElement. If an optional argument is passed, it will
+            // force the class to be added or removed based on the truthiness of
+            // that value.
+            // 
+            // Return `this`, for chaining.
+            toggle: {
+                value: function(bool) {
+                    if (typeof bool !== 'undefined') {
+                        this.statusElement.classList.toggle(this.statusVisibleClass, bool);
```

Comment:
A quick test (in Firefox anyway) appears as though you can always do this, regardless of whether `bool` is set or not.